### ISSUE TITLE
Testing Basset-style Basenji

### ISSUE THREAD

**Issue by htdashti** [calico/basenji#106]
Hi, Im trying to follow the example instruction for running Basset-style peak prediction. I went through the steps and didnt change the input or the parameter file. After the basenji_train.py was completed and running the basenji_test.py I got an unexpected Test AUROC of 0.55407. I would appreciate your suggestions on what I did wrong.

This is the tail of the test report: 
basenji_test.py train_basset/params.json train_basset/model_best.h5 data_basset
...
model_strides [192]
target_lengths [1]
target_crops [3]
2021-12-20 11:52:00.006469: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)
2021-12-20 11:52:00.025889: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 3299990000 Hz
18714/18714 [==============================] - 3843s 205ms/step - loss: 0.0757 - pearsonr: nan - r2: -inf

Test Loss: 0.07572
Test AUROC: 0.55407
Test AUPRC: 0.29810
 
Thanks!

**Comment by davek44** [comment]
Hi, I have been able to replicate this mediocre performance. (I don't see the "nan" and "-inf" from your logs.) I'm not sure what the problem is, but I'll be working to chase it down.

**Comment by davek44** [comment]
Quick update--I believe the problem is in my bespoke implementation of multi-task AUC statistics, which inherit from the TensorFlow AUC class. My code works on v2.4 but breaks on v2.6, due to some changes they made. I'll work on fixing that. But in the meantime, you might consider benchmarking your model using scikit-learn or other AUROC/AUPRC statistics, because it might be working much better than my metrics suggest.

**Comment by davek44** [comment]
Indeed, I tracked down the bug to my SeqAUC class. I've updated it to the latest Tensorflow version, and dodged a weird enum bug. I can now train these Basset models to normal high AUROC.

**Comment by htdashti** [comment]
Thanks!
